### PR TITLE
Create ui-c8y-1019-15-1-add-missing-widgets-to-device-management-application.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by bode

Ticket: [DM-3266](https://cumulocity.atlassian.net/browse/DM-3266)
Original PR: [5977](https://github.softwareag.com/IOTA/cumulocity-ui/pull/5977)

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1019-15-1-add-missing-widgets-to-device-management-application.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Add missing widgets to Device management application
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1019-15-1-add-missing-widgets-to-device-management-application.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Device management application now includes all available widgets
```

File: content/change-logs/device-management/ui-c8y-1019-15-1-add-missing-widgets-to-device-management-application.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3266
+version: 1019.15.1
+---
+Add missing widgets to Device management application
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1019-15-1-add-missing-widgets-to-device-management-application.md:17]: The Claude AI suggested to use the following body:
```suggestion
The device management application was missing some widgets that were available in other applications. With this change, all available widgets have now been added to the device management application. This allows users to access and utilize the full range of widget functionality directly within the device management application, providing a more comprehensive and convenient experience for managing their devices.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/device-management/ui-c8y-1019-15-1-add-missing-widgets-to-device-management-application.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Device management application now includes all available widgets
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1019-15-1-add-missing-widgets-to-device-management-application.md:3]:
```suggestion
title: Device Management application now includes all available widgets
```

**Review by BeateRixen — APPROVED** [review]

File: content/change-logs/device-management/ui-c8y-1019-15-1-add-missing-widgets-to-device-management-application.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  - value: change-VSkj2iV9m
+    label: Fix
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3266
+version: 1019.15.1
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1019-15-1-add-missing-widgets-to-device-management-application.md:15]:
```suggestion
version: 10.19.15.1
```

**Review by l2p- — APPROVED** [review]

**Pull request merged**